File: GeneradorCodigo/Python/Multiplicar.py
import Suma


def Multiplicar(A, B):
	aux = 0
	print(aux)
	for i in range(A):
		aux = Suma(aux,int(B))
		print(aux)

	return aux


print("Introducir el valor para A:")
A = input()
print("Introducir el valor para B:")
B = input()
C = Multiplicar(int(A), int(B))
print("El resultado es: ", C)
H=input()

File: Func/Multiplicar.c
#include <stdio.h>
#include <stdlib.h>
#include "Multiplicar.h"


int main(int argc, char *argv[]){
	int A, B, C;
	printf("Introducir el valor para A:");
	scanf("%d", &A);
	printf("Introducir el valor para B:");
	scanf("%d", &B);
	C = Multiplicar(A, B);
	printf("El resultado es: %d", C);
}